helpers/slots.js

```diff
@@ -71,8 +68,7 @@ module.exports = {
 			epochTime = this.getTime();
 		}
 
-		var d = constants.EPOCH_TIME;
-		var t = Math.floor(d.getTime() / 1000) * 1000;
+		var t = Math.floor(EPOCH_TIME.getTime() / 1000) * 1000;
```

Comment:
Same here, would be nice to rename this variable.

Reply:
I removed the variable here as well